.github/pull_request_template.md

```diff
@@ -1,23 +1,79 @@
 <!--
+A few suggestions about filling out this PR
 
-Replace this comment with a description of what's being changed by this PR.
+1. Use a descriptive title for the PR.
+2. If this pull request is work in progress, create a draft PR instead of prefixing the title with WIP.
+3. Please label this PR with one of the following labels, depending on the scope of your change:
+- New feature
+- Bug fix
+- Enhancement
```

Comment:
It might be unclear to some users what the difference between a `New feature` and an `Enhancement` is. Perhaps we should add some explanatory text?